src/converters/piral-ng/README.md

```diff
@@ -166,17 +166,17 @@ Depending on your Angular needs you'd want to share more packages.
 
 Depending on the mounted component different services are injected. the following table lists the names of the injected services per component type.
 
-| Component | Props            | Piral   | Context   |
-|-----------|------------------|---------|-----------|
-| Tile      | `TileProps`      | `Piral` | `Context` |
-| Page      | `PageProps`      | `Piral` | `Context` |
-| Modal     | `ModalProps`     | `Piral` | `Context` |
-| Extension | `ExtensionProps` | `Piral` | `Context` |
-| Menu      | `MenuProps`      | `Piral` | `Context` |
+| Component | Props   | Piral   | Context   |
+|-----------|---------|---------|-----------|
+| Tile      | `Props` | `Piral` | `Context` |
+| Page      | `Props` | `Piral` | `Context` |
+| Modal     | `Props` | `Piral` | `Context` |
+| Extension | `Props` | `Piral` | `Context` |
+| Menu      | `Props` | `Piral` | `Context` |
 
 To use such a service the `@Inject` decorator should be used with the explicit name.
 
-The following code snippet illustrates the injection of the `TileProps` service into a sample tile component.
+The following code snippet illustrates the injection of the `Tile Props` service into a sample tile component.
```

Comment:
Maybe have it rephrased as "...injection of the `Props` service from an `TileProps` interface into a sample tile component."

Reply:
Done